designsafe/static/scripts/data-depot/components/data-depot-nav/data-depot-nav.component.html

```diff
@@ -17,11 +17,11 @@
           <a uib-tooltip="{{ childItem.description }}" class="list-group-item" ui-sref="{{ childItem.state }}" ui-sref-active="active">
             {{ childItem.name }}
           </a>
+          <div ng-If="childItem.name == 'Shared with Me'"> <hr> </div>
+          </div>
```

Comment:
Looks like this line doesn't need to be added, while you can keep line #24 to keep the formatting aligned.